templates/nginx_cfg.tmpl

```diff
@@ -25,6 +25,7 @@ server {{
         proxy_cache {NGINX_KEYS_ZONE};
         proxy_cache_use_stale {NGINX_CACHE_USE_STALE};
         proxy_cache_valid {NGINX_CACHE_VALID};
+        {NGINX_CACHE_ALL}
```

Comment:
Would you need a `;` at the end of the line as above?

Reply:
the variable has the ; already, if cache_ all is activated an empty line will be in place.

Reply:
Given how it works for the other scenarios, this doesn't seem obvious